react-components/src/apps/details.js

```diff
@@ -1,7 +1,45 @@
+<<<<<<< HEAD
 import CategoryTree from './details/CategoryTree';
 import ToolDetails from './details/ToolDetails';
 
 export {
     CategoryTree,
     ToolDetails,
+=======
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import CategoryTree from './details/CategoryTree';
+import CopyTextArea from './details/CopyTextArea';
+import ToolDetails from './details/ToolDetails';
+
+const renderCategoryTree = (elementID, app, presenter, appearance) => {
+    ReactDOM.render(
+        <CategoryTree app={app} presenter={presenter} appearance={appearance} />,
+        document.getElementById(elementID)
+    );
+};
+
+const renderCopyTextArea = (elementID, btnText, textToCopy) => {
+    ReactDOM.render(
+        <CopyTextArea btnText={btnText} text={ textToCopy }/>,
+        document.getElementById(elementID)
+    );
+};
+
+const renderToolDetails = (elementID, toolDetailsAppearance, app) => {
+    ReactDOM.render(
+        <ToolDetails appearance={toolDetailsAppearance} app={app} />,
+        document.getElementById(elementID)
+    );
+};
+
+export {
+    CategoryTree,
+    CopyTextArea,
+    ToolDetails,
+    renderCategoryTree,
+    renderCopyTextArea,
+    renderToolDetails
+>>>>>>> 471d1ebd7... Add ToolDetails and CategoryTree React integration.
```

Comment:
Yep, it looks like you need to run another `git rebase --interactive master` but remove the first 2 commits of mine that were already merged by my PR:

* Add ToolDetails and CategoryTree React integration.
* Add npm install, build, and clean tasks to gradle scripts.

Reply:
Sounds good. Will do that!